test/LibraryManager.Test/Providers/JsDelivr/JsDelivrProviderTest.cs

```diff
@@ -222,23 +222,24 @@ public async Task InstallAsync_WithGlobPatterns_CorrectlyInstallsAllMatchingFile
         }
 
         [TestMethod]
-        public void GetSuggestedDestination()
+        public void GetSuggestedDestination_NullLibrary_ReturnsEmptyString()
         {
-
             Assert.AreEqual(string.Empty, _provider.GetSuggestedDestination(null));
+        }
 
+        [DataTestMethod]
+        [DataRow("jquery", "jquery")]
+        [DataRow("@angular/cli", "angular/cli")]
+        public void GetSuggestedDestination(string libraryName, string expected)
+        {
             var library = new JsDelivrLibrary()
             {
-                Name = "jquery",
+                Name = libraryName,
                 Version = "3.3.1",
```

Comment:
>"3.3.1 [](start = 26, length = 6)

Nit: Should this be a part of the DataRow data or removed if it's not needed/doesn't matter? Same for UnpkgProviderTest

Reply:
Doesn't matter, it only looks at the name.